packages/cli/src/commands/deviceprofiles/presentation.ts

```diff
@@ -0,0 +1,44 @@
+import { DeviceProfile, PresentationDevicePresentation } from '@smartthings/core-sdk'
+
+import { ListingOutputAPICommand } from '@smartthings/cli-lib'
+
+import { buildTableOutput } from '../presentation'
+
+
+export default class ProfilePresentationCommand extends ListingOutputAPICommand<PresentationDevicePresentation, DeviceProfile> {
+	static description = 'get the presentation associated with a device profile'
+
+	static flags = ListingOutputAPICommand.flags
+
+	static args = [{
+		name: 'id',
+		description: 'device profile UUID or the number of the profile from list',
+	}]
+
+	primaryKeyName = 'id'
+	sortKeyName = 'name'
+
+	protected buildTableOutput(presentation: PresentationDevicePresentation): string {
+		return buildTableOutput(presentation, this.tableGenerator)
+	}
+
+	async run(): Promise<void> {
+		const { args, argv, flags } = this.parse(ProfilePresentationCommand	)
+		await super.setup(args, argv, flags)
+
+		this.processNormally(
+			args.id,
+			() => { return this.client.deviceProfiles.list() },
+			async (id) => {
+				const profile = await this.client.deviceProfiles.get(id)
+				if (profile.metadata) {
+					return this.client.presentation.getPresentation(profile.metadata.vid, profile.metadata.mnmn)
+				} else {
+					this.logger.error('No presentation defined for device profile')
```

Comment:
You should be able to just `throw Error('No presentation...` here and this will log the error and exit for you.